assets/css/jquery-gritter/jquery.gritter.css

```diff
@@ -23,71 +23,71 @@
     left: 20px;
 }
 .gritter-item-wrapper {
-	position:relative;
-	margin:0 0 10px 0;
-	background:url('../images/ie-spacer.gif'); /* ie7/8 fix */ 
+    position:relative;
+    margin:0 0 10px 0;
+    background:url('../../images/ie-spacer.gif'); /* ie7/8 fix */ 
 }
 .gritter-top {
-	background:url(../images/gritter.png) no-repeat left -30px;
-	height:10px;
+    background:url(../../images/gritter.png) no-repeat left -30px;
```

Comment:
Thought we were keeping all dependencies for js librariies withing that same directory.  css stuff was for things like webkit, fontawesome and the like.  Got no problem with anything, but it should be as simple as possible.